lib/tool_shed/galaxy_install/tool_dependencies/recipe/step_handler.py

```diff
@@ -157,33 +158,11 @@ def url_download( self, install_dir, downloaded_file_name, download_url, extract
             If the checksum does not match an exception is thrown.
         """
         file_path = os.path.join( install_dir, downloaded_file_name )
-        src = None
-        dst = None
-
-        # Set a timer so we don't sit here forever.
-        start_time = time.time()
         try:
-            src = urllib2.urlopen( download_url )
-            dst = open( file_path, 'wb' )
-            while True:
-                chunk = src.read( basic_util.CHUNK_SIZE )
-                if chunk:
-                    dst.write( chunk )
-                else:
-                    break
-                time_taken = time.time() - start_time
-                if time_taken > basic_util.NO_OUTPUT_TIMEOUT:
-                    err_msg = 'Downloading from URL %s took longer than the defined timeout period of %.1f seconds.' % \
-                        ( str( download_url ), basic_util.NO_OUTPUT_TIMEOUT )
-                    raise Exception( err_msg )
+            download_to_file( download_url, file_path, chunk_size=basic_util.CHUNK_SIZE)
```

Comment:
No, the old code used `basic_util.NO_OUTPUT_TIMEOUT` as a time limit for the whole download (start to finish), while the timeout inside `download_to_file()` is for [blocking operations like the connection attempt](https://docs.python.org/2/library/urllib2.html).